src/test/java/com/pba/authservice/integration/ActiveUserDaoIntegrationTest.java

```diff
@@ -0,0 +1,177 @@
+package com.pba.authservice.integration;
+
+import com.pba.authservice.exceptions.AuthDaoException;
+import com.pba.authservice.mockgenerators.ActiveUserMockGenerator;
+import com.pba.authservice.persistance.model.ActiveUser;
+import com.pba.authservice.persistance.repository.ActiveUserDao;
+import jakarta.validation.constraints.NotNull;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.springframework.test.context.jdbc.Sql;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.util.Optional;
+import java.util.UUID;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+@Testcontainers
+@Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = "classpath:/cleanup.sql")
+public class ActiveUserDaoIntegrationTest {
```

Comment:
create a class called BaseIntegrationTest where you have all the common annotations and code in there, including the container